[2023-04-13T13:40:16.831+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T13:40:09.827115+00:00 [queued]>
[2023-04-13T13:40:16.842+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.inject_data manual__2023-04-13T13:40:09.827115+00:00 [queued]>
[2023-04-13T13:40:16.844+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T13:40:16.845+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-13T13:40:16.846+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-13T13:40:16.860+0000] {taskinstance.py:1309} INFO - Executing <Task(SqliteOperator): inject_data> on 2023-04-13 13:40:09.827115+00:00
[2023-04-13T13:40:16.865+0000] {standard_task_runner.py:55} INFO - Started process 1954 to run task
[2023-04-13T13:40:16.868+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'inject_data', 'manual__2023-04-13T13:40:09.827115+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpi7zk3_se']
[2023-04-13T13:40:16.869+0000] {standard_task_runner.py:83} INFO - Job 169: Subtask inject_data
[2023-04-13T13:40:16.937+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.inject_data manual__2023-04-13T13:40:09.827115+00:00 [running]> on host ffea11fb791c
[2023-04-13T13:40:17.046+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=inject_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-13T13:40:09.827115+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-13T13:40:09.827115+00:00
[2023-04-13T13:40:17.049+0000] {sql.py:254} INFO - Executing: 
        INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
        (1681392968,
        287.66,
        45,
        "scattered clouds",
        1.71);
        
[2023-04-13T13:40:17.059+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T13:40:17.246+0000] {base.py:73} INFO - Using connection ID 'measure_db' for task execution.
[2023-04-13T13:40:17.250+0000] {sql.py:375} INFO - Running statement: 
        INSERT INTO measures_ext (timestamp, temp, humidity, cloudiness, wind_speed) VALUES
        (1681392968,
        287.66,
        45,
        "scattered clouds",
        1.71);
        , parameters: None
[2023-04-13T13:40:17.267+0000] {sql.py:384} INFO - Rows affected: 1
[2023-04-13T13:40:17.290+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=inject_data, execution_date=20230413T134009, start_date=20230413T134016, end_date=20230413T134017
[2023-04-13T13:40:17.327+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-13T13:40:17.345+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
